backend/fms_core/migrations/0065_v4_12_0.py

```diff
@@ -1,43 +1,64 @@
-import reversion
+from django.conf import settings
 from django.db import migrations
 from django.contrib.auth.models import User
+import reversion
 
-ADMIN_USERNAME = 'biobankadmin'
+from fms_core.models import StepOrder, SampleNextStep, Step, Protocol, StepSpecification, SampleNextStepByStudy, Workflow, Study
```

Comment:
Do not import models into the migration this can lead to circular import which the migration system really do not like ... You should use the dynamic model loading used in most migrations (exemple:  PropertyType = apps.get_model("fms_core", "PropertyType").